### PR TITLE
feat: Add a command to change DA with an existing rollapp

### PR DESCRIPTION
**Pull request by ItayLevyOfficial**
# PR Standards

## Opening a pull request should be able to meet the following requirements

---

For Author:

- [x]  Targeted PR against correct branch
- [x]  included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [x]  Linked to Github issue with discussion and accepted design
- [x]  Targets only one github issue
- [ ]  Wrote unit and integration tests
- [ ]  All CI checks have passed
- [ ]  Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code)

---

For Reviewer:

- [ ]  confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ]  Reviewers assigned
- [ ]  confirmed all author checklist items have been addressed

---

After reviewer approval:

- [ ]  In case targets main branch, PR should be squashed and merged.
- [ ]  In case PR targets a release branch, PR should be rebased.

### CODE REVIEW

File: cmd/config/set/da.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+		return err
+	}
+	rlpCfg.DA = newDa
+	if err := sequencer.UpdateDymintDAConfig(rlpCfg); err != nil {
+		return err
+	}
+	return config.WriteConfigToTOML(rlpCfg)
+}
+
+func cleanDADir(cfg config.RollappConfig) error {
```
Comment (mtsitrin) [inline cmd/config/set/da.go:39]: won't fail when updating from mock? mock has da dir?

File: cmd/config/set/da.go [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+	return os.RemoveAll(filepath.Join(cfg.Home, consts.ConfigDirName.DALightNode))
```
Comment (mtsitrin) [inline cmd/config/set/da.go:40]: will remove the old keys
open a todo issue to add warning or require `-force` flag when deleting keys

**Review by mtsitrin — APPROVED** [review]

File: cmd/config/set/da.go (hunk 1 above)
Comment (ItayLevyOfficial) [reply cmd/config/set/da.go:39]: removeAll return nil when the dir does not exist, so it is ok.

File: cmd/config/set/da.go (hunk 2 above)
Comment (ItayLevyOfficial) [reply cmd/config/set/da.go:40]: opened, #454

**Pull request merged**